### PR TITLE
Build Tools by Ivan Genchev

### PR DESCRIPTION
**Pull request by freekman**
1)Ant task

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <javac srcdir="${src.dir}" destdir="builder" classpathref="classpath"/>
+  </target>
+
+  <target name="package" description="creates dir target">
+    <mkdir dir="target"/>
+  </target>
+
+  <target name="jarWithoutJarJar" depends="compile" description="without jar jar">
+    <jar jarfile="target/sample.jar">
+      <fileset dir="builder"/>
```
Comment (mgenov) [inline build.xml]: build is more common name

File: build.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="Ant Project wihout jar jar" basedir=".">
+  <property name="src.dir" value="./src"/>
```
Comment (mgenov) [inline build.xml]: why ./?

File: build.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  <path id="classpath">
+    <fileset dir="lib" includes="**/*.jar"/>
+  </path>
+
+  <target name="compile">
+    <javac srcdir="${src.dir}" destdir="builder" classpathref="classpath"/>
```
Comment (mgenov) [inline build.xml]: why src directory is with property definition but the target is not ?

File: build.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <fileset dir="builder"/>
+      <zipfileset src="lib/log4j-1.2.15.jar"/>
+      <manifest>
+        <attribute name="Main-Class" value="com.clouway.Main"/>
+      </manifest>
+    </jarjar>
+  </target>
+
+  <target name="run">
+    <java jar="target/sample.jar" fork="true"/>
```
Comment (mgenov) [inline build.xml]: what fork actually does ?

Comment (freekman) [inline build.xml]: If set to true triggers the class execution on another VM

**Comment by mgenov** [comment]
LGTM !

**Comment by freekman** [comment]
Should i merge then

**Comment by mgenov** [comment]
Yeah, you already have LGTM !

**Pull request merged**